### ISSUE TITLE
since today: HTTP Error 415: Unsupported Media Type /content/ead/clevel

### ISSUE THREAD

**Issue by u-jung** [ArchivesPortalEuropeFoundation/ape-backend#1]
Hello,

Yesterday it still worked fine. Today I get a HTTP Error 415: Unsupported Media Type when I try to get https://api.archivesportaleurope.net/services/content/ead/clevel/C410412582 (example).
Yesterday's header has 'content-type': 'application/vnd.ape-v4+json'  ('accept' didn't work)
I also tried with 'content-type': 'application/vnd.ape-v5+json' but I got an  HTTP Error 400: Bad Request

Regards.
U. Jung (30.8.17 13h18 UTC)

**Comment by sowrov** [comment]
I think you are missing API key. Take a look here: http://www.archivesportaleurope.net/information-api;jsessionid=9AB4C7F8CB2506DBB3EF78F56A72C8BA

**Comment by u-jung** [comment]
Thanks for the answer. Indeed I have an API key (which ends with ..f11dcc). As I already wrote, yesterday the same key worked fine. Even when I changed the API key I noted the same trouble

But meanwhile I've got **the solution**:
- The 415 Error was caused by the content-type which changed over night from 'application/vnd.ape-v4+json' to 'application/vnd.ape-v5+json'
- The 400 Error was caused by an syntax error inside my script. 

The script works fine now.

Conclusion:
The WIki page http://wiki.archivesportaleurope.net/index.php/APE_API_documentation need to be changed within the Versioning section:
From _'accept' : 'application/vnd.ape-v4+json'_ to 'content-type' : _'application/vnd.ape-v5+json'_

Regards.

U. Jung
